pkg/packagekit/updates.jsx

```diff
@@ -1312,7 +1312,12 @@ class OsUpdates extends React.Component {
                     // data looks like:
                     // downloading\tbash-completion;1:2.6-1.fc26;noarch;updates-testing
                     // updating\tbash-completion;1:2.6-1.fc26;noarch;updates-testing
-                const pkgs = { _time: Date.parse(timeSpec) };
+                const _time = Date.parse(timeSpec);
+                if (isNaN(_time)) {
+                    debug(`Transaction has an invalid timespec=${timeSpec}`);
+                    return;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22575-ce26b778-20251107-101628-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)